helm-charts/strimzi-kafka-operator-helm3/crds/040-Crd-kafka.yaml

```diff
@@ -0,0 +1,3686 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kafkas.kafka.strimzi.io
+  labels:
+    component: kafkas.kafka.strimzi.io-crd
+spec:
+  group: kafka.strimzi.io
+  versions:
+  - name: v1beta1
+    served: true
+    storage: true
+  - name: v1alpha1
+    served: true
+    storage: false
+  version: v1beta1
+  scope: Namespaced
+  names:
+    kind: Kafka
+    listKind: KafkaList
+    singular: kafka
+    plural: kafkas
+    shortNames:
+    - k
+  additionalPrinterColumns:
+  - name: Desired Kafka replicas
+    description: The desired number of Kafka replicas in the cluster
+    JSONPath: .spec.kafka.replicas
+    type: integer
+  - name: Desired ZK replicas
+    description: The desired number of ZooKeeper replicas in the cluster
+    JSONPath: .spec.zookeeper.replicas
+    type: integer
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
```

Comment:
In the README, you've said a minimum Kubernetes version of 1.9+. 

In https://github.com/strimzi/strimzi-kafka-operator/pull/2232#issuecomment-557465149 @tombentley was suggesting that having this top-level `type: object` value is only supported from 1.12 onwards.